src/components/v5/common/ActionSidebar/hooks/useGetExpenditureData.ts

```diff
@@ -0,0 +1,16 @@
+import { useGetExpenditureQuery } from '~gql';
+
+export const useGetExpenditureData = (
+  expenditureId: string | null | undefined,
+) => {
+  const { data, loading } = useGetExpenditureQuery({
+    variables: {
+      expenditureId: expenditureId || '',
+    },
+  });
```

Comment:
The query shouldn't just run with an empty id, please make sure to specify `skip` option to avoid unnecessary requests